src/platform/forms-system/test/js/state/helpers.unit.spec.js

```diff
@@ -255,6 +255,50 @@ describe('Schemaform formState:', () => {
       expect(newSchema.properties.field['ui:collapsed']).to.be.true;
       expect(newSchema).not.to.equal(schema);
     });
+    it('should not set collapsed on expandUnder field with function condition based on formData', () => {
+      const schema = {
+        type: 'object',
+        properties: {
+          field: {},
+          field2: {},
+        },
+      };
+      const uiSchema = {
+        field: {
+          'ui:options': {
+            expandUnder: 'field2',
+            expandUnderCondition: (condition, formData) =>
+              formData.field2 === 'bleh',
```

Comment:
I'd prefer to see these two tests refactored so it's obvious that they're two sides of the same test, but at the least, could you drop a comment here to indicate that this condition is the difference?